sample-conf.yaml

```diff
@@ -215,3 +215,12 @@ hashmail:
 prometheus:
   enabled: true
   listenaddr: "localhost:9000"
+
+# Console and file logger settings.
+logging:
+  console:
+    disable: false
+    call-site: short
+  file:
+    disable: true
+    call-site: long
```

Comment:
yaml parser is struggling with these 🙃

Reply:
Could be reserved words. Try `'short' and 'long' instead?

Reply:
it isnt just the call-site. it is anything from that unexported empedded struct. only works for the 'style' option when compiled in 'dev' mode. 

seems to be a yaml issue (cause things work for the lnd.conf file just fine)

Reply:
fixed here: https://github.com/lightningnetwork/lnd/pull/9576